framework_tvb/tvb/tests/framework/core/services/project_service_test.py

```diff
@@ -296,11 +296,11 @@ def test_project_disk_size(self):
             assert '0.0 KiB' != project.disk_size_human
 
             prj_folder = self.structure_helper.get_project_folder(project)
-            actual_disk_size = self.compute_recursive_h5_disk_usage(prj_folder)[0]
+            actual_disk_size = FilesHelper.compute_recursive_h5_disk_usage(prj_folder)[0]
 
             ratio = float(actual_disk_size) / project.disk_size
             msg = "Real disk usage: %s The one recorded in the db : %s" % (actual_disk_size, project.disk_size)
-            assert ratio < 1.7, msg
+            assert ratio < 1.4, msg
```

Comment:
Is this the minimum ratio?
I wonder where from we have differences left.

Reply:
I don't think it is the minimum, I just reverted to the ratio we used to check. I could make it smaller.